MediaEditor.xcodeproj/xcshareddata/xcschemes/MediaEditor.xcscheme

```diff
@@ -28,6 +28,24 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "8B50471723D7D32B00068F66"
+               BuildableName = "Tests.xctest"
+               BlueprintName = "Tests"
+               ReferencedContainer = "container:MediaEditor.xcodeproj">
+            </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "MediaEditorHubTests/testDoNotShowActivityIndicatorIfImageIsNotBeingLoaded()">
+               </Test>
+               <Test
+                  Identifier = "MediaEditorHubTests/testShowActivityIndicatorWhenSwipingToAnImageBeingLoaded()">
+               </Test>
+            </SkippedTests>
```

Comment:
Whilst skipping these is a temporary tech debt incurrence, I'd like to suggest we use `XCTExpectFailure` instead.

I opened #29 for this and added a rationale.

---

TIL that the noun deriving from the verb _incur_ is _incurrence_, not _incursion_. However, "tech debt incursion" might still be appropriate ⚔️ ⚔️ ⚔️

Reply:
> tech debt incursion
😱 😂